Demo/Demo/Models/CreateOrderParams.swift

```diff
@@ -15,3 +16,30 @@ struct Amount: Codable {
     let currencyCode: String
     let value: String
 }
+
+struct Vault: Codable {
+
+    let storeInVault: String
+}
+
+
+struct Attributes: Codable {
+
+    let vault: Vault
+    let customer: CardVaultCustomer?
+}
+
+struct CardVaultCustomer: Codable {
+
+    let id: String
+}
+
+struct VaultCard: Codable {
+
+    let attributes: Attributes
+}
+
+struct VaultCardPaymentSource: Codable {
+
+    let card: VaultCard
+}
```

Comment:
Styling suggestion - IMO it's more readable to define these in the order that they appear in the struct above

```suggestion

struct VaultCardPaymentSource: Codable {

    let card: VaultCard
}

struct VaultCard: Codable {

    let attributes: Attributes
}

struct Attributes: Codable {

    let vault: Vault
    let customer: CardVaultCustomer?
}

struct Vault: Codable {

    let storeInVault: String
}

struct CardVaultCustomer: Codable {

    let id: String
}
```

Reply:
Another nitpick - use `Encodable` instead of `Codable` since we never will decode this model type

Reply:
Addressed in 8e95c5d1aff1212b258d840f60f056c4359c5c18 (reordering in CreateOrderParam along with merge conflict resolution)